config/locales/letters_en.yml

```diff
@@ -0,0 +1,157 @@
+        <div><strong>Prisoner’s income</strong></div>
+        <p>Prisoner Income and Expenditure Statement (PIES) showing earnings, spending, private cash and savings.</p>
+        <p>If you’re in a private prison, your statement needs to show earnings and private cash.</p>
+        <div><strong>Any other monthly income</strong></div>
+        <p>Documentation of any other income that you’re receiving. </p>
+        <p>Please send this letter back with the information about your income so we can match it to your application.</p> 
+        <p>Yours sincerely,</p>
+        <p>%{user_name}</p>
+      no_evidence_received_html: |
+        <p>Dear %{full_name}</p>
+        <p>We wrote to you about your application for help with fees, asking to see evidence of your income by %{evidence_date}.</p>  
+        <p>As we haven’t received payment, we’re unable to process your application. This may have an impact on the original case or claim related to this application.</p> 
+        <p>You will need to make a new application if you still require help.</p>
+        <p>Yours sincerely,</p>
+        <p>%{user_name}</p>
+      evidence_incorrect_html: |
+        <p>Dear %{full_name}</p>
+        <p>We have received the additional information requested for your application for help with fees. However, there’s a problem with the documents you have sent.</p>  
+        <p>(PLEASE EXPLAIN WHY EVIDENCE CAN'T BE ACCEPTED, FOR EXAMPLE: BANK STATMENTS ARE OUT OF DATE</p>
+        <p> Unfortunately we’re unable to process your application.</p>
+        <p> If you still want help with your fees, you will have to complete a new application. </p>
+        <p>Yours sincerely,</p>
+        <p>%{user_name}</p>
+    part-payment:
+      send_part_payment_html: |
+        <p>Dear %{full_name}</p>
+        <p><strong>Reference: %{reference}</strong></p>
+        <p>We have received your application for help with fees. You are eligible to make a part-payment and must pay %{part_payment_amount} towards the fee.</p> 
+        <p>We’ve calculated this as follows:</p> 
+        <p>Your total monthly income: %{total_income}</p>
+        <p>Number of children: %{children}</p>
+        <p>Amount to pay: %{part_payment_amount}</p>
+        <br>
+        <p><strong>How to pay</strong></p> 
+        <ul> 
+          <li>by post with a cheque written out to ‘HM Courts and Tribunals Service’ for the amount</li>
+          <li>by debit card over the phone</li> 
+          <li>in person with a cheque, cash or debit card at the court counter</li>
+        <br> 
+        <p>Please make sure you pay this amount by %{date_for_part_payment}</p>
+        <br>
+        <p><strong>If you don’t agree with the amount you have to pay</strong></p>  
+        <p>If you disagree with our decision about the amount you have to pay, you can appeal to %{name_delivery_manager}, Delivery Manager at %{court_tribunal_name}. You need to do this by %{date_for_appeal}.</p>  
```

Comment:
@leighmoney this was the only section I couldn't further match.  %{name_delivery_manager}, %{court_tribunal_name} and %{date_for_appeal} are not currently stored/generated, so I set the text above...